contact.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
```

Comment:
This file should be renamed from `contact.html` to `contact/index.html` so that we don't have filename extensions in the URL (They look unprofessional).